certified-connectors/Redque/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3548 @@
+                "CONTAINER_CREATION_FAILED",
+                "CONTAINER_NOT_FOUND",
+                "CONTAINER_NOT_REMOVED",
+                "DOCUMENT_NOT_RETURNED",
+                "DOCUMENT_RETURNED_NOT_SAVED",
+                "DOCUMENT_NOT_READY",
+                "DOCUMENT_UNSUPPORTED_TYPE",
+                "DOCUMENT_EXTRACT_RESULT_NOT_FOUND",
+                "DOCUMENT_FILE_NOT_FOUND",
+                "DOCUMENT_FILE_UPLOAD_FAILED",
+                "DOCUMENT_FORBIDDEN_TYPE",
+                "ATTACHMENT_NOT_ATTACHED",
+                "PASSWORD_NOT_ACCEPTED",
+                "EMAIL_NOT_FOUND",
+                "NO_TEXT_DETECTED",
+                "ENTITY_DELETE_FAILED"
+            ],
+            "type": "string"
+        },
+        "SecureSocketOptions": {
+            "description": "Secure socket options.",
+            "enum": [
+                "None",
+                "Auto",
+                "SslOnConnect",
+                "StartTls",
+                "StartTlsWhenAvailable"
+            ],
+            "type": "string"
+        },
+        "StringApiFilterEqual": {
+            "description": "Equal filter implementation",
+            "type": "object",
+            "properties": {
+                "value": {
+                    "type": "string"
+                }
+            }
+        },
+        "StringApiFilterMultipleChoice": {
+            "description": "Multiple choice filter implementation",
+            "type": "object",
+            "properties": {
+                "values": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                }
+            }
+        },
+        "StringApiFilterNotEqual": {
+            "description": "Equal filter implementation",
+            "type": "object",
+            "properties": {
+                "value": {
+                    "type": "string"
+                }
+            }
+        },
+        "StringApiListValueUpdate": {
+            "type": "object",
+            "properties": {
+                "value": {
+                    "type": "string"
+                },
+                "operation": {
+                    "$ref": "#/definitions/ApiListValueOperation"
+                }
+            }
+        },
+        "StringApiValue": {
+            "description": "Generic response value",
+            "type": "object",
+            "properties": {
+                "value": {
+                    "description": "Value",
+                    "type": "string"
+                }
+            }
+        },
+        "StringApiValueUpdate": {
+            "type": "object",
+            "properties": {
+                "value": {
+                    "type": "string"
+                }
+            }
+        }
+    },
+    "x-ms-connector-metadata": [
+        {
+            "propertyName": "Website",
+            "propertyValue": "https://redque.com"
```

Comment:
@jirisodomka the website URL should be a valid URL. please change to "https://redque.cz/en/" or "https://redque.cz"